tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/KafkaHelper.cs

```diff
@@ -353,5 +353,18 @@ internal static void TryInjectHeaders<TTopicPartitionMarker, TMessage>(
                 Log.Warning(ex, "There was a problem injecting headers into the Kafka record. Disabling Headers injection");
             }
         }
+
+        internal static void DisableHeadersIfUnsupportedBroker(Exception exception)
+        {
+            if (_headersInjectionEnabled && exception is not null && exception.Message.Contains("Unknown broker error"))
```

Comment:
Ignore case just to be safe.
```suggestion
            if (_headersInjectionEnabled && exception is not null && exception.Message.Contains("Unknown broker error", StringComparison.OrdinalIgnoreCase))
```

Reply:
Thanks!

Reply:
I used the `exception.Message.IndexOf("Unknown broker error", StringComparison.OrdinalIgnoreCase) != -1` representation as we don't always have access to the `Contains` overload with the string comparators.